randovania/gui/widgets/multiplayer_session_users_widget.py

```diff
@@ -95,7 +100,7 @@ class MultiplayerSessionUsersWidget(QtWidgets.QTreeWidget):
     _last_session: MultiplayerSessionEntry | None = None
     _session: MultiplayerSessionEntry
     _user_widgets: dict[int, UserWidgetEntry]
-    _world_widgets: dict[uuid.UUID, WorldWidgetEntry]
+    _world_widgets: dict[tuple[int | None, uuid.UUID], WorldWidgetEntry]  # (user id, world UUID): worldWidgetEntry
```

Comment:
don't know what you mean here. as an extra attribute for MultiplayerSessionUsersWidget? or do you mean to just have it as a type alias, so that you can do `dict[tuple[UserID, uuid.UUID], ...]`?

Reply:
The later